packages/components/src/components/section/section.lite.tsx

```diff
@@ -1,37 +1,21 @@
-import {
-	onMount,
-	useDefaultProps,
-	useMetadata,
-	useRef,
-	useStore
-} from '@builder.io/mitosis';
-import { DEFAULT_ID } from '../../shared/constants';
-import { cls, uuid } from '../../utils';
-import { DBSectionProps, DBSectionState } from './model';
+import { useDefaultProps, useMetadata, useRef } from '@builder.io/mitosis';
+import { cls } from '../../utils';
+import { DBSectionProps } from './model';
 
 useMetadata({});
 useDefaultProps<DBSectionProps>({});
 
 export default function DBSection(props: DBSectionProps) {
 	const _ref = useRef<HTMLDivElement | any>(null);
-	// jscpd:ignore-start
-	const state = useStore<DBSectionState>({
-		_id: DEFAULT_ID
-	});
-
-	onMount(() => {
-		state._id = props.id || 'section-' + uuid();
-	});
-	// jscpd:ignore-end
 
 	return (
 		<section
 			ref={_ref}
-			id={state._id}
-			class={cls('db-section', props.className)}
+			id={props.id}
+			class={cls('db-section', props.className, props.id)}
```

Comment:
```suggestion
			class={cls('db-section', props.className)}
```

@michaelmkraus {props.id} was only included for debugging, wasn't it ?